### DISCUSSION TITLE
How can we host the tunnel mole service on our server and enable the https

### DISCUSSION THREAD

**Discussion by jeevan2nani** [robbie-cahill/tunnelmole-client#49] [Q&A]
Can you guys help me out how to host the tunnelmole service on my server and enable https ?

**Comment by robbie-cahill** [comment]
I've only ever done this on the main service but unfortunately did not document the steps. I'm sure I could do it for a new domain with several hours of work, but unfortunately don't have the spare time at the moment.

I used a Lets Encrypt certificate and an nginx reverse proxy server which sits in front of the tunnelmole service, so my method involved more than just the service alone.
